src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Persona/BitPersonaTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Persona
+{
+    [TestClass]
+    public class BitPersonaTests : BunitTestContext
+    {
+        [DataTestMethod,
+            DataRow("secondaryText", "TertiaryText", "Text", "OptionalText"),
+            DataRow("secondaryText", "TertiaryText", "Text", "OptionalText"),
+            DataRow("secondaryText", "TertiaryText", "Text", "OptionalText"),
+            DataRow("secondaryText", "TertiaryText", "Text", "OptionalText")]
+        public void BitPersonaShouldCurrectWorkText(string secondaryText, string tertiaryText, string primarytext, string optionalText)
```

Comment:
Plz, change the order of the parameters. Text, SecondaryText, TertiaryText, OptionalText